README.md

```diff
@@ -1,6 +1,175 @@
+- [Contributing](#contributing)
+- [Community](#community)
+- [Versioning](#versioning)
+- [Creators](#creators)
+- [Thanks](#thanks)
+- [Copyright and license](#copyright-and-license)
+
+## [](#quick-start)Quick start
+
+Several quick start options are available:
+
+- [Docker](#setup-with-docker)
+- [Python Virtual Environment](#setup-with-python-virtual-environment)
+
+## [](#setup-with-docker)Setup with Docker
+
+#### Dependencies
+
+- [Docker](https://docs.docker.com/engine/installation/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
+
+### Installation
+
+Run the following commands:
+
+```bash
+git clone https://github.com/aichner/Wagtail-Template.git
+cd Wagtail-Template
+docker-compose up --build -d
+docker-compose up
+```
+
+The demo site will now be accessible at [http://localhost:8000/](http://localhost:8000/).
+
+**Important:** This `docker-compose.yml` is configured for local testing only, and is _not_ intended for production use.
+
+### Debugging
+
+To tail the logs from the Docker containers in realtime, run:
+
+```bash
+docker-compose logs -f
+```
+
+## [](#setup-with-python-virtual-environment)Setup with Python Virtual Environment
+
+You can start a Wagtail project from this template without setting up Docker and simply use a virtual environment,
+which is the [recommended installation approach](https://docs.python.org/3/library/venv.html) for all Python projects itself.
+
+#### Dependencies
+
+- Python 3.5, 3.6 or 3.7
+
+### Installation
+
+With [PIP](https://github.com/pypa/pip) installed, run:
+
+    git clone https://github.com/aichner/Wagtail-Template.git
+    cd Wagtail-Template
+    python --version
+    python -m pip --version
+
+Confirm that this is showing a compatible version of Python 3.x. If not, and you have multiple versions
+of Python installed on your system, you may need to specify the appropriate version when creating the venv:
+
+    python3 -m venv /path/to/new/virtual/environment
+
+Once a virtual environment has been created, it can be “activated” using a script in the virtual environment’s
+binary directory. The invocation of the script is platform-specific (<venv> must be replaced by the path of the
+directory containing the virtual environment):
+
+| Platform | Shell      | Command to activate virtual environment |
+| :------- | :--------- | :-------------------------------------- |
+| Posix    | bash/zsh   | \$ source <venv>/bin/activate           |
+|          | fish       | \$ . <venv>/bin/activate.fish           |
+|          | csh/tcsh   | \$ source <venv>/bin/activate.csh       |
+| Windows  | cmd.exe    | C:\> <venv>\Scripts\activate.bat        |
+|          | PowerShell | PS C:\> <venv>\Scripts\Activate.ps1     |
+
+Now we're ready to set up the project itself:
+
+    pip install -r requirements/base.txt
+
+To set up your database and load initial data, run the following commands:
+
+    ./manage.py migrate
+    ./manage.py runserver
+
+## [](#status)Status
```

Comment:
This heading with its certain content makes no sense in this repository.

Reply:
yikes xD